pkg/plugins/optional/config/v1alpha/constants.go

```diff
@@ -0,0 +1,22 @@
+/*
+Copyright 2022 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package v1alpha
+
+// nolint: lll
+const MetadataDescription = `This command will add Controller Manager Config file to the project:
+	('config/manager/controller_manager_config.yaml')
```

Comment:
Why do we need this file?
Could we follow the same structure used/adopt in other plugins and keep the const where it will be used?